README.md

```diff
@@ -1,2 +1,137 @@
-# Webpack-demo
-A Microverse exercise. In this exercise, I will build a simple yet powerful webpack boilerplate, which I can later use as a starting point in my projects.
+<a name="readme-top"></a>
+
+<div align="center">
+  <br/>
+
+  <h1><b>
+LeaderboardList App
+</b></h1>
+<p>
+In this activity I will set up a JavaScript project for the Leaderboard list app, using webpack and ES6 features, notably modules. I will develop a first working version of the app following a wireframe, but without styling - just functionality. In the following activities (milestone 2 and 3), I will consume the Leaderboard API using JavaScript async and await and add some styling. </p> <hr>
+<p>
+The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service. </p>
+<br/>
+
+</div>
+
```

Comment:
- [ ] Based on the template this project description should be below the table of contents based on the "About the project" section. Kindly move this section to its proper place.